.travis.yml

```diff
@@ -0,0 +1,49 @@
+---
+sudo: required
+services:
+  - docker
+language: python
+python:
+  - "2.7"
+env:
+  global:
+    - OS_NAME=centos
+  matrix:
+    - OS_MAJOR_VERSION=6
+    - OS_MAJOR_VERSION=7
+before_install:
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt update
+  - sudo apt install -y docker-ce
+  - sudo apt install -y cppcheck
+  - docker --version
+install:
+  - wget http://archive.ubuntu.com/ubuntu/pool/universe/r/rpmlint/rpmlint_1.7-1_all.deb
+  - sudo dpkg -i rpmlint_1.7-1_all.deb || sudo apt install -f -y
+  - gem install mdl
+script:
+  # First syntax, linting and pre-test stages
+  - mdl -s relaxed README.md
+  - rpmlint nagios-plugins-bdii.spec
+  # Lint C file
+  - cppcheck src/check_bdii_entries.c
+  # Then, start the build container
+  - docker run --name build_container -d -ti -v $PWD:/tmp -w /tmp $OS_NAME:$OS_MAJOR_VERSION /bin/bash -c 'while true ; do sleep 1000 ; done'
+  # Now, install prerequisites for the build
+  - docker exec -i -w /root build_container yum install -y rpm-build
+  - docker exec -i build_container yum-builddep -y nagios-plugins-bdii.spec
+  # Eventually build the rpm
+  - docker exec -i -w /tmp build_container make rpm
+  # and install it
+  - docker exec -i -w /tmp/build/RPMS/x86_64 build_container bash -c "yum localinstall -y nagios-plugins-bdii*.el${OS_MAJOR_VERSION}.x86_64.rpm"
```

Comment:
I think you need to add the EPEL repo otherwise some rpm deps cannot be installed

Reply:
yep yep :)